From 5fc2b4e68ed5d1838feefab22e474517d8240408 Mon Sep 17 00:00:00 2001
From: Mike Lothian <mike@fireburn.co.uk>
Date: Thu, 11 Jul 2024 18:21:54 +0100
Subject: [PATCH] Try the next render node before returning

On PRIME systems usually renderD129 is the onboard graphics and renderD128 is the discrete GPU, allow the loop to continue to the next node
---
 media/gpu/vaapi/vaapi_wrapper.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/gpu/vaapi/vaapi_wrapper.cc b/media/gpu/vaapi/vaapi_wrapper.cc
index 7885871dbd3ceb..6d56dff4910f3e 100644
--- a/media/gpu/vaapi/vaapi_wrapper.cc
+++ b/media/gpu/vaapi/vaapi_wrapper.cc
@@ -1524,7 +1524,7 @@ void VADisplayStateSingleton::PreSandboxInitialization() {
       continue;
     }
     va_display_state.drm_fd_ = base::ScopedFD(drm_file.TakePlatformFile());
-    return;
+    continue;
   }
 }
 
